packages/react-core/src/components/CalendarMonth/examples/CalendarMonthDefault.tsx

```diff
@@ -1,4 +1,12 @@
 import React from 'react';
 import { CalendarMonth } from '@patternfly/react-core';
 
-export const CalendarMonthDefault: React.FunctionComponent = () => <CalendarMonth date={new Date()} />;
+export const CalendarMonthDefault: React.FunctionComponent = () => {
```

Comment:
Re: this comment from Titani back when I made an update to the component: https://github.com/patternfly/patternfly-react/pull/7082#issuecomment-1079408467

Do we still want the "Default" example to be static, and then just update the "Selectable date" example to showcase this new callback?

Reply:
I think that the default example starting with no date selected, but still allowing a date to be selected is more in line with how most of our component examples work - even just the basic/default examples? I dont think we have many non-interactive react examples for components which are built for interaction.

Reply:
@tlabaj ^